### PR TITLE
etesync-dav: fix dependencies

### LINKED ISSUE #178385: etesync-dav fails to run with ModuleNotFoundError on 22.05 and unstable
### Describe the bug
etesync-dav fails to run with ModuleNotFoundError on 22.05 and unstable

### Steps To Reproduce
1. `nix-shell --pure -p etesync-dav` --run etesync-dav

### Expected behavior
I expect the etesync-dav service to run

### Screenshots
```
 $ nix-shell --pure -p etesync-dav --run etesync-dav
Traceback (most recent call last):
  File "/nix/store/056cblm38l6bvagz8v088qb9yxdmlq0x-etesync-dav-0.32.1/bin/.etesync-dav-wrapped", line 11, in <module>
    from etesync_dav.manage import Manager
  File "/nix/store/056cblm38l6bvagz8v088qb9yxdmlq0x-etesync-dav-0.32.1/lib/python3.9/site-packages/etesync_dav/manage.py", line 23, in <module>
    from .radicale.creds import Credentials
  File "/nix/store/056cblm38l6bvagz8v088qb9yxdmlq0x-etesync-dav-0.32.1/lib/python3.9/site-packages/etesync_dav/radicale/__init__.py", line 16, in <module>
    from . import web, storage, rights  # noqa: F401
  File "/nix/store/056cblm38l6bvagz8v088qb9yxdmlq0x-etesync-dav-0.32.1/lib/python3.9/site-packages/etesync_dav/radicale/web.py", line 15, in <module>
    import pkg_resources
ModuleNotFoundError: No module named 'pkg_resources'
```

### Additional context
This works on 21.11 but not 22.05 of nixos-unstable

### Notify maintainers
@Valodim 

### Metadata

```console
 - system: `"x86_64-linux"`
 - host os: `Linux 5.15.46, NixOS, 22.05 (Quokka), 22.05.926.8b538fcb329`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.8.1`
 - channels(aidenn): `""`
 - channels(root): `"nixos-22.05.926.8b538fcb329, nixos-unstable-22.05pre358986.062a0c5437b"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

### PR DESCRIPTION
**Pull request by jasom**
setuptools and pysocks are both needed for this to work.

Newer WTForms is also needed for this to work; the flask-wtf tests fail with the newer WTForms, so I also disabled the tests for flask-wtf.

Reported issue with flask-wtf tests failing upstream as etesync/etesync-dav#268

Fixes #178385

###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by mfossen** [comment]
LGTM, tested on NixOS unstable.

**Comment by alapshin** [comment]
This works for me on NixOS unstable.

We also have https://github.com/NixOS/nixpkgs/pull/178358 that addresses same problems

**Comment by thyol** [comment]
##### Reviewed points

- [x] package build on x86_64
- [x] executables tested on x86_64

**Comment by thyol** [comment]
@ofborg eval

File: pkgs/applications/misc/etesync-dav/default.nix [hunk 1]
```diff
@@ -13,6 +13,7 @@ let
           inherit version;
           sha256 = "ff177185f891302dc253437fe63081e7a46a4e99aca61dfe086fb23e54fff2dc";
         };
+        checkPhase = ":";
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/etesync-dav/default.nix:16]:
```suggestion
        doCheck = false;
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by thyol** [comment]
@jasom A patch for the proposed changes. d3d0dad0f648caa59377fd3bca9d7cfad2949ad6

@SuperSandro2000 Under which circumstance should we update the python werkzeug package for everybody?

**Comment by SuperSandro2000** [comment]
Werkzeug is already at version 2.2.2, so that shouldn't be a problem.

@ofborg eval

**Comment by thyol** [comment]
This is at the moment broken because of https://github.com/etesync/etebase-py/issues/38

**Comment by Alch-Emi** [comment]
The mentioned issue has since been closed, is it possible to move forward with this?  I'd do it myself, but I'm struggling to get my dev environment set up for working on the main `nixpkgs` repo ;-;

**Pull request closed without merging**